.github/workflows/scheduled.yml

```diff
@@ -73,10 +73,8 @@ jobs:
             testing_formulae+=("${formulae[${index}]}")
           done
 
-          json="$(
-            brew info --json=v2 "${testing_formulae[@]}" |
-              jq --compact-output '[.formulae[] | select(.deprecated or .disabled | not) | .name]'
-          )"
+          jq_filter='[.formulae[] | select(.deprecated or .disabled | not) | { name, curl: (.urls.stable.using == "homebrew_curl")}]'
+          json="$(brew info --json=v2 "${testing_formulae[@]}" | jq --compact-output "${jq_filter}")"
```

Comment:
This is fine but I think it'd be really nice to push this pretty involve `jq` logic into Ruby at some point.